executor/runtime/docker/docker_ssh.go

```diff
@@ -103,6 +103,14 @@ func addContainerSSHDConfig(c runtimeTypes.Container, tw *tar.Writer, cfg config
 }
 
 func addContainerSSHDConfigWithData(c runtimeTypes.Container, tw *tar.Writer, cfg config.Config, caData []byte) error {
+	if err := tw.WriteHeader(&tar.Header{
+		Name:     "titus/etc/ssh",
```

Comment:
Probably doesn't matter, but the `Name:` in the old code below has a leading `/` but `Name:` here does not.

Reply:
The "/" gets stripped off by the Tar implementation.